lib/rules/tuple.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+/**	Signature: function(value, field, parent, errors, context)
+ */
+module.exports = function ({ schema, messages }, path, context) {
+	const src = [];
+
+	if (schema.items != null) {
+		if (!Array.isArray(schema.items)) {
+			throw new Error(`Invalid '${schema.type}' schema. The 'items' field must be an array.`);
+		}
+
+		if (schema.items.length === 0) {
+			throw new Error(`Invalid '${schema.type}' schema. The 'items' field must not be an empty array.`);
+		}
+	}
+
+	src.push(`
+		if (!Array.isArray(value)) {
+			${this.makeError({ type: "tuple", actual: "value", messages })}
+			return value;
+		}
+
+		var len = value.length;
+	`);
+
+	src.push(`
```

Comment:
I don't think we should limit this new rule that it can't be empty. It's same as array, so it should be an `empty` property.

https://github.com/icebob/fastest-validator/blob/master/lib/rules/array.js#L17